scripts/unlock-packages.js

```diff
@@ -1,30 +1,33 @@
 #!/usr/bin/env node
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-const fs = require('fs');
 
-/**
- * Remove specific @cloudscape-design/* packages where we should always use the latest minor release
- */
-const filename = require.resolve('../package-lock.json');
-const packageLock = require(filename);
+const fs = require('node:fs');
+const lockfiles = ['package-lock.json'];
```

Comment:
Even if this array looks redundant, it is easier to keep this way to simplify pulling this script updates from the upstream: https://github.com/cloudscape-design/components/blob/main/scripts/unlock-package-lock.js